micro-s3/readme.md

```diff
@@ -65,25 +65,8 @@ public class DataLoader  implements ScheduledJob<Job>{
 }
 
 @Component
-public class DataCleaner  implements ScheduledJob<Job>{
-	
-	private final ManifestComparator<String> comparator;
-	@Autowired
-	public  DataCleaner(ManifestComparator comparator) {
-		this.comparator = comparator.<String>withKey("test-key");
-	}
-	@Override
-	public SystemData<String,String> scheduleAndLog() {
-		try{
-		        comparator.cleanAll();
-			return SystemData.<String,String>builder().errors(0).processed(1).build();
-		}catch(Exception e){
-			return SystemData.<String,String>builder().errors(1).processed(0).build();
-		}
-		
-	}
 
-}
+\lyvyv vv; ;r;;;;'' res'''e'e''lhlhlllllgm       xdvl;;l;l; 
```

Comment:
Yes, formatting seems all over the place too. Perhaps Eclipse is using tabs not spaces again, I'll check reformat and remove.